ui/raidboss/raidboss_config.js

```diff
@@ -1029,7 +1029,7 @@ UserConfig.registerOptions('raidboss', {
       name: {
         en: 'Time to recolor timer as expiring soon (seconds)',
         de: 'Zeit bis ein bald auslaufender Timer umgefärbt wird (in Sekunden)',
-        fr: 'Durée de recolorisation du timer avant expiration (secondes)',
+        fr: 'Durée avant la recolorisation du temps qui expire bientôt (secondes)',
```

Comment:
You should keep _avant expiration_ because _qui expire bientôt_ is ugly.

Reply:
La première, je ne la comprend pas non plus. 
1: Durée avant la recolorisation du temps du temps avant expiration. (bientôt) ?
2: Durée de la recolorisation du temps avant expiration. (bientôt) ?
3:  Recolorisation du temps avant expiration.  (bientôt) ?
Si rien ne convient, je n'ai peut être pas compris le sens, et je m'en excuse, et  je laisserai tel quel.

Reply:
C'est le fait de recoloriser la barre de temps avant son expiration, donc on pourrait mettre :
_Recolorisation de la barre de temps avant expiration_, si ça te convient.

Reply:
Carrément o/